front/src/main/home.jsx

```diff
@@ -9,14 +9,20 @@ import operationalStudiesImg from 'assets/pictures/home/operationalStudies.svg';
 import rollingStockEditorImg from 'assets/pictures/home/rollingstockeditor.svg';
 import stdcmImg from 'assets/pictures/home/stdcm.svg';
 import proudLogo from 'assets/proud-logo-osrd-color-white.svg';
+import xmasLogo from 'assets/xmas-logo-osrd-color-white.svg';
 import Card from 'common/BootstrapSNCF/CardSNCF/CardSNCF';
 import { ModalProvider } from 'common/BootstrapSNCF/ModalSNCF/ModalProvider';
 import NavBarSNCF from 'common/BootstrapSNCF/NavBarSNCF';
 
 export default function Home() {
   const { t } = useTranslation('home/home');
 
-  const logo = new Date().getMonth() === 5 ? proudLogo : defaultLogo;
+  let logo = defaultLogo;
+  if (new Date().getMonth() === 5) {
+    logo = proudLogo;
+  } else if (new Date().getMonth() === 11) {
+    logo = xmasLogo;
+  }
```

Comment:
How about doing this so we don't mixup the numbers (5 is June, really?).

```suggestion
  // 0-based months
  const JUNE = 5;
  const DECEMBER = 11;
  if (new Date().getMonth() === 5) {
    logo = proudLogo;
  } else if (new Date().getMonth() === 11) {
    logo = xmasLogo;
  }
```

Reply:
In the rest of the front, we would do something like this (you can move the const outside the `Home` function) : 

```suggestion
  const MONTH_VALUES = {
    JUNE: 5,
    DECEMBER: 11
  }
  
  if (new Date().getMonth() === MONTH_VALUES.JUNE) {
    logo = proudLogo;
  } else if (new Date().getMonth() === MONTH_VALUES.DECEMBER) {
    logo = xmasLogo;
  }
```